### DISCUSSION TITLE
Screenshot difference due to page scroll

### DISCUSSION THREAD

**Discussion by JulioPazini** [FRSOURCE/cypress-plugin-visual-regression-diff#287] [Q&A] [answered]
I'm having this problem where sometimes the screenshot is a few pixels off because of the scroll of the page. Here's the images for comparison:

**Original image:**
![FAQ snapshot #0](https://github.com/FRSOURCE/cypress-plugin-visual-regression-diff/assets/41480304/3303c4de-2eee-4ff6-9f37-fd787f6681a8)

**Image after running the test:**
![FAQ snapshot #0 actual](https://github.com/FRSOURCE/cypress-plugin-visual-regression-diff/assets/41480304/d7c248b0-c088-408d-9cb2-6a64ca22e30c)

**Diff between them:**
![FAQ snapshot #0 diff](https://github.com/FRSOURCE/cypress-plugin-visual-regression-diff/assets/41480304/4f2c13c5-ad49-4f2a-9b95-06eb342cc05e)

Sometimes it passes without a problem. Does anybody know a way to make sure the screenshot is always taken in the same place? or maybe I'm missing something?

**Comment by FRSgit** [comment, marked as answer]
Hey!

Do you still have a reproduction scenario for this one?

As far as I remember, whenever I've encountered a case like this I was able to force scroll position using cypress apis - like scrollTo. Have you tried that?
